backend/src/main/java/com/ody/mate/dto/request/MateStatusRequest.java

```diff
@@ -0,0 +1,20 @@
+package com.ody.mate.dto.request;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.annotation.Nullable;
+
+public record MateStatusRequest(
+
+        @Schema(description = "위치추적 불가 여부", example = "false")
+        boolean isMissing,
+
+        @Schema(description = "현재 위도", example = "126.234524")
+        @Nullable
+        String currentLatitude,
+
+        @Schema(description = "현재 경도", example = "39.123345")
+        @Nullable
+        String currentLongitude
```

Comment:
[필수]

위도, 경도 example이 반대네요~
아래처럼 반대로 바꿔주시면 좋을 것 같아요!
```suggestion
        @Schema(description = "현재 위도", example = "39.123345")
        @Nullable
        String currentLatitude,

        @Schema(description = "현재 경도", example = "126.234524")
        @Nullable
        String currentLongitude
```

Reply:
<img width="362" alt="image" src="https://github.com/user-attachments/assets/f510d5ec-fd1a-44e5-8e66-8bc716659973">
API 문서에 작성되어 있는 Request 바디 그대로 작성했습니다 !

Reply:
API docs가 잘못 되어있네요..!
docs는 제가 수정해놓겠습니다!
코드는 수정해주셔야 할 것 같아요~

Reply:
[수정완료]